### PR TITLE
Remove links to https://community.neptune.ai/ in docs

### PR DESCRIPTION
**Pull request by pzurowski**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -82,7 +82,6 @@ Discover Neptune
 - |Example Project|: See how example project looks in Neptune
 - |YouTube channel|: Provides hands-on videos that showcase key Neptune features.
 - |Neptune blog|: Provides in-depth articles about best practices in machine learning experimentation (among other things)
-- |Neptune community|: Meet other Neptune users and developers and start a discussion.
```
Comment (kamil-kaczmarek) [inline docs/index.rst:85]: also drop link definition at the bottom of the file.

File: docs/execution-environments/index.rst [hunk 2]
```diff
@@ -91,10 +91,6 @@ Various Notebook flavours:
...
 
-.. |forum| raw:: html
-
-    <a href="https://community.neptune.ai/c/feature-requests" target="_blank">feature request</a>
-
 .. |neptune-client| raw:: html
 
-    <a href="/api-reference/neptune/index.html" target="_blank">neptune-client</a>
\ No newline at end of file
+    <a href="/api-reference/neptune/index.html" target="_blank">neptune-client</a>
```
Comment (kamil-kaczmarek) [inline docs/execution-environments/index.rst:96]: you can drop this link, as it's not used anywhere.

File: docs/integrations/pytorch_lightning.rst [hunk 3]
```diff
@@ -613,9 +613,29 @@ You may also like these two integrations:
 
     <a href="https://github.com/neptune-ai/neptune-examples/blob/master/integrations/pytorch-lightning/docs/Neptune-PyTorch-Lightning-basic.py" target="_blank">GitHub</a>
 
-.. |forum| raw:: html
+.. |PyTorch| raw:: html
 
-    <a href="https://community.neptune.ai/" target="_blank">forum</a>
+    <a href="https://docs.neptune.ai/integrations/pytorch.html" target="_blank">PyTorch</a>
+
+.. |PyTorch Ignite| raw:: html
```
Comment (kamil-kaczmarek) [inline docs/integrations/pytorch_lightning.rst:620]: You can drop all links that you added here: pytorch, ingnite, catalyst, skorch. optuna, plotly.

We use sphinx's tags and references, to link between pages.

**Review by kamil-kaczmarek — CHANGES_REQUESTED** [review]

File: docs/execution-environments/index.rst (hunk 2 above)
Comment (pzurowski) [reply docs/execution-environments/index.rst]: **done**

File: docs/index.rst (hunk 1 above)
Comment (pzurowski) [reply docs/index.rst:85]: **done**

File: docs/integrations/pytorch_lightning.rst (hunk 3 above)
Comment (pzurowski) [reply docs/integrations/pytorch_lightning.rst]: **done**

**Review by kamil-kaczmarek — APPROVED** [review]
Looks good

**Pull request merged**